cmd/genji/shell/shell.go

```diff
@@ -189,7 +189,7 @@ func (sh *Shell) executeInput(in string) error {
 	switch {
 	// if it starts with a "." it's a command
 	// it must not be in the middle of a multi line query though
-	case strings.HasPrefix(in, ".") && sh.query == "":
+	case strings.HasPrefix(in, "."):
```

Comment:
I am not sure to understand why did you remove the `&& sh.query == ""` statement. Could you elaborate please?

Reply:
I did that because `.indexes` could take an argument `.indexes tableName` that led me to change the behaviour of`runCommand` by checking the len of the splitted input string. 
`sh.query == ""` was a 'guard' against input command with arguments but now we need it for `.indexes`.